### PR TITLE
Turning IPMI off for unknown products

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

**Comment by tboerger** [comment]
Please squash the commits. This is just a port from our patch on the packages. :+1: after squashing the commits.

**Comment by MaximilianMeister** [comment]
+1

File: chef/cookbooks/ipmi/recipes/ipmi-discover.rb [hunk 1]
```diff
@@ -30,7 +30,14 @@
   end
 end
 
-unsupported = [ "KVM", "Bochs", "VMWare Virtual Platform", "VMware Virtual Platform", "VirtualBox" ]
+unsupported = [
+  "KVM",
+  "Bochs",
+  "VMWare Virtual Platform",
+  "VMware Virtual Platform",
```
Comment (tboerger) [inline chef/cookbooks/ipmi/recipes/ipmi-discover.rb]: I have not seen that before but this line is a duplicate

Comment (vuntz) [inline chef/cookbooks/ipmi/recipes/ipmi-discover.rb]: It's not; look at the case of w (vs W)

Comment (tboerger) [inline chef/cookbooks/ipmi/recipes/ipmi-discover.rb]: Oh yeah, i looked 2 times at it and i have not seen that :D

**Pull request merged**